SecretAgent/SecretAgent/Screen/Siren/TestSirenViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  TestSirenViewController.swift
+//  SecretAgent
+//
+//  Created by Minkyeong Ko on 2022/11/21.
+//
+
+import AVFoundation
+import UIKit
+
+import SnapKit
+
+class TestSirenViewController: BaseViewController {
+    // MARK: - Properties
+
+    private var sirenPlayer: AVAudioPlayer?
```

Comment:
p4.

```suggestion
    private var sirenPlayer: AVAudioPlayer!
```

아래에 sirenPlayer를 호출하기 전에 할당될 것이 보장되니 다음과 같이 선언하고 밑에 옵셔널로 처리하지 않는게 더 좋다고 생각해요 :) 꼭 필요하지 않으면 옵셔널로 만들 필요가 없다고 생각해서요! 다니의 의견이 궁금합니다!